src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreResult.cs

```diff
@@ -76,6 +76,11 @@ public class RestoreResult : IRestoreResult
         /// </summary>
         private PackagesLockFile NewPackagesLockFile { get; }
 
+
+        private string DependencyGraphSpecFilePath { get; }
+
+        private DependencyGraphSpec DependencyGraphSpec { get; }
```

Comment:
Delete extra blank line.

Use readonly fields instead of private properties.

Reply:
Private properties are readonly automatically. 
Any concerns with that over readonly field?

Reply:
fixed